system/handlers/admin/SysConfig.cfc

```diff
@@ -149,6 +160,42 @@ component extends="preside.system.base.AdminHandler" {
 		setNextEvent( url=event.buildAdminLink( linkTo="sysconfig.category", queryString="id=#categoryId#" ) );
 	}
 
+	public void function configHistory( event, rc, prc ) {
+		var categoryId = Trim( rc.id   ?: "" );
+		var siteId     = Trim( rc.site ?: "" );
+
+		try {
+			prc.category = systemConfigurationService.getConfigCategory( id = categoryId );
+		} catch( "SystemConfigurationService.category.notFound" e ) {
+			event.notFound();
+		}
+	}
+
+	public void function getConfigHistoryForAjaxDataTables( event, rc, prc ) {
+		prc.setting = structKeyList( event.getCollectionForForm( "system-config.#rc.id#" ) );
+
+		var nextVersionNumber = presideObjectService.getNextVersionNumber();
```

Comment:
See other review comment about avoiding this. This method call inserts a new version number in the version number table. Instead, we shoud calculate the real max version number for the settings category.